backend/routes/quests.go

```diff
@@ -103,3 +118,100 @@ func GetTodayStartTime(w http.ResponseWriter, r *http.Request) {
 
 	WriteDataJson(w, "\""+string(todayStartTime.UTC().Format(time.RFC3339))+"\"")
 }
+
+func SendChat(w http.ResponseWriter, r *http.Request) {
+	userAddress := r.URL.Query().Get("address")
+	if userAddress == "" {
+		WriteErrorJson(w, http.StatusBadRequest, "Missing address parameter")
+		return
+	}
+
+	var chat FactionChat
+	err := json.NewDecoder(r.Body).Decode(&chat)
```

Comment:
The body should just contain the factionKey and message, so other fields can be removed from the `FactionChat` type here.

Also, please use the `ReadJsonBody` utility from `routes/requests.go`